firmware/include/OrbsWiFiManager.h

```diff
@@ -0,0 +1,19 @@
+#ifndef ORBS_WIFI_MANAGER_H
+#define ORBS_WIFI_MANAGER_H
+
+#include "OrbsWiFiManagerConfig.h"
+#include <WiFiManager.h>
+
+/**
+ * We override WiFiManager in order to enable "_showBack" (show back buttons on WebPortal pages).
```

Comment:
According to the comments by the upstream author in the GH issue tracker, it's a deliberate decision to not include a setter yet (because it's considered experimental)